src/main/java/org/alliancegenome/curation_api/model/document/es/ModelDocument.java

```diff
@@ -0,0 +1,43 @@
+package org.alliancegenome.curation_api.model.document.es;
+
+import com.fasterxml.jackson.annotation.JsonView;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import org.alliancegenome.curation_api.model.entities.AffectedGenomicModel;
+import org.alliancegenome.curation_api.model.entities.ConditionRelation;
+import org.alliancegenome.curation_api.model.entities.Organization;
+import org.alliancegenome.curation_api.model.entities.ontology.DOTerm;
+import org.alliancegenome.curation_api.view.View;
+
+import java.util.*;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+@JsonView(View.ModelDocumentView.class)
+public class ModelDocument extends ESDocument {
```

Comment:
Ideally this would be called GeneModelDocumen

Reply:
Better would be: AffectedGenomicModelDocument as that's what this is.

Reply:
I think, AffectedGenomicModelDocument would be more in line with it.